tools-local/tasks/installer.tasks/RegenerateDownloadTable.cs

```diff
@@ -173,7 +173,9 @@ void AddLink(string name, string type)
 
             AddLink("Runtime-Deps", "runtime-deps");
             AddLink("Host", "host");
```

Comment:
Side note (no action required): Even if we have both CoreCLR and Mono runtimes here, the hosts or targeting packs would be identical between the two.